packages/scripts/index.js

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env node
+
+const chalk = require('chalk');
+const fs = require('fs');
+const loadConfigFile = require('rollup/dist/loadConfigFile');
+const ora = require('ora');
+const path = require('path');
+const process = require('process');
+const rollup = require('rollup');
+
+const getRollupConfigPath = (file) => {
+  if (file && !fs.existsSync(path.resolve(path.join('.', file)))) {
+    // If the user explicitly provides a rollup config file and if it is not available, then throw an error and exit.
+    console.error(`Could not find ${path.join('.', file)}`);
+    process.exit(1);
+  }
+  // If user does't provide a rollup config file, then search for `rollup.config.js` in the project directory. If it is
+  // not found, then use `basePluginRollupConfig.js` which contains pretty reasonable defaults.
+  const projectRollupConfigFilePath = path.resolve(path.join('.', file || 'rollup.config.js'));
+  return fs.existsSync(projectRollupConfigFilePath)
+    ? projectRollupConfigFilePath
+    : path.resolve(path.join(__dirname), 'basePluginRollupConfig.js');
+};
+
+const OUTPUT_DIR_REGEX = /\/?(\w+)$/;
+
+const getOutputDir = (output) => {
+  const outputDirMatch = OUTPUT_DIR_REGEX.exec(output);
+  return outputDirMatch.length > 1 ? outputDirMatch[1] : output;
+};
+
+const startHandler = ({ file }) => {
+  process.env.ROLLUP_WATCH = true;
+
+  loadConfigFile(getRollupConfigPath(file))
+    .then(async ({ options, warnings }) => {
```

Comment:
in this case I'd prefer to either lean into async/await or use promise chaining, but not mix them